hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/AutoCreationLogic.java

```diff
@@ -98,10 +99,9 @@ public boolean reclaimPendingAliases() {
                 final var alias = syntheticTxnBody.getAlias();
                 if (!alias.isEmpty()) {
                     aliasManager.unlink(alias);
-                }
-                final var evmAddress = syntheticTxnBody.getEvmAddress();
-                if (!evmAddress.isEmpty()) {
-                    aliasManager.unlink(evmAddress);
+                    if (alias.size() != EVM_ADDRESS_SIZE) {
```

Comment:
⚠️ Should this be `==` instead of `!=`?

Reply:
I believe it is correct:
1. If the alias is an `ED`/`ECDSA` key alias --- we unlink it @ line 100, then try to (_possibly_) unlink the evm address derived from it  @ line 103 ( `forgetEvmAddress` fails silently)
3. If the alias is an `evm address` --- we unlink it @ line 100, and we do not go into this if,

Reply:
This does read in a very confusing way! It would probably have been best to rename the confusing `forgetEvmAddress(alias)` method which actually takes in an alias of type ED/EC key, but I just added an inline comment below the if statement for now.

Reply:
Aha, got it. Thanks guys. 👍